src/patterns/BidSnapshot/BidSnapshot.tsx

```diff
@@ -0,0 +1,147 @@
+ *
+ * [Storybook Link](https://phillips-seldon.netlify.app//?path=/docs/patterns-bidsnapshot--overview)
+ */
+const BidSnapshot = forwardRef<HTMLDivElement, BidSnapshotProps>(
+  (
+    {
+      activeBid,
+      auctionStatus = AuctionStatus.ready,
+      bids = [],
+      bidsLabelText = bidsTranslation,
+      children,
+      className,
+      closingText = 'Closes in',
+      currentBidText = 'Current bid',
+      lang = 'en',
+      lotCloseDate,
+      startingBidText = 'Starting bid',
+      soldForText = 'Sold for',
+      wonForText = 'Won for',
+      ...props
+    },
+    ref,
+  ) => {
+    const { className: baseClassName, ...commonProps } = getCommonProps(props, 'BidSnapshot');
+    const currentDate = new Date();
+    return (
+      <div {...commonProps} className={classnames(baseClassName, className)} {...props} ref={ref}>
+        {
+          // auctionStatus is not past (auction is active or upcoming),
+          !(auctionStatus === AuctionStatus.past) ? (
+            // if auction has not started
+            auctionStatus === AuctionStatus.ready ? (
+              // show starting bid
+              <DetailList hasSeparators className={`${baseClassName}__text`}>
+                <Detail label={startingBidText} value={bids[0]} hasWrap={false} />
+              </DetailList>
+            ) : (
+              // otherwise, show current bid
+              <DetailList hasSeparators className={`${baseClassName}__text`}>
+                <Detail
+                  label={currentBidText}
+                  subLabel={bids.length > 0 && `(${bidsLabelText(bids.length)})`}
+                  value={bids[bids.length - 1]}
+                  hasWrap={false}
+                />
+              </DetailList>
+            )
+          ) : // if auctionStatus is past,
+          // if activeBid was the last bid, show won for activeBid
+          activeBid === bids[bids.length - 1] ? (
+            <DetailList hasSeparators className={`${baseClassName}__text`}>
+              <Detail label={wonForText} value={bids[bids.length - 1]} hasWrap={false} />
+            </DetailList>
+          ) : (
+            // if active bid was not last bid show Sold for
+            <DetailList hasSeparators className={`${baseClassName}__text`}>
+              <Detail label={soldForText} value={bids[bids.length - 1]} hasWrap={false} />
+            </DetailList>
+          )
+        }
+
+        {activeBid && auctionStatus !== AuctionStatus.ready ? children : null}
+
+        {
+          // Countdown timer
+          lotCloseDate && lotCloseDate > currentDate ? (
+            <Countdown
+              endDateTime={lotCloseDate}
+              label={closingText}
+              variant={CountdownVariants.compact}
+              locale={SupportedLanguages[lang]}
+              formatDurationStr={(str) => str.replace(/seconds?/, 'sec').replace(/minutes?/, 'min')}
```

Comment:
doesn't this need to handle translations for the different locales?

Reply:
our other location in short but for future translations we may want to. We can surface this prop to the Bidsnapshot component and let the user pass in the function.

Reply:
I added this as a prop for BidSnapshot so it can be passed in at implementation. Defaulting to the strings transitions for english as seen in the design.